frontend/src/hocs/withHostCommunication/withHostCommunication.tsx

```diff
@@ -44,6 +44,19 @@ export interface HostCommunicationHOC {
    */
   onModalReset: () => void
 
+  /**
+   * Callback to be called when the Streamlit app rerun a script.
+   */
```

Comment:
Nits: Perhaps adjust comments as follows?
- `Callback to be called when the Streamlit app's host requests to rerun the script`
- `Callback to be called when the Streamlit app's host requests to stop the script`
- `Callback to be called when the Streamlit app's host requests to clear the cache`

Reply:
thanks for the suggestion!

the reason why I wrote these comments to be in sync with exiting callbacks above and below. 

I think they are better at explaining current behavior because e.g. `onScriptRerun` callback would be executed on any rerun, not just for a rerun triggered by the host frame message.